ui/src/pages/project.tsx

```diff
@@ -144,13 +153,14 @@ export const Project = ({
           <div className='flex items-center gap-8 text-gray-500 text-sm mr-2'>
             {Object.entries(tabs).map(([key, value]) => (
               <Tooltip key={key} title={value.label}>
-                <div className={classNames('cursor-pointer', { 'text-blue-500': tab === key })}>
-                  <FontAwesomeIcon
-                    icon={value.icon}
-                    onClick={() => {
-                      navigate(generatePath(tabs[key as keyof typeof tabs]?.path ?? '', { name }));
-                    }}
-                  />
+                <div
+                  className={classNames('cursor-pointer', { 'text-blue-500': tab === key })}
+                  onClick={() => {
+                    navigate(generatePath(tabs[key as keyof typeof tabs]?.path ?? '', { name }));
+                  }}
+                >
+                  <FontAwesomeIcon icon={value.icon} className='mr-2' />
+                  {value.label}
```

Comment:
As I already started working on some changes here, I would avoid too many changes to solve the same problem.
https://github.com/akuity/kargo/issues/3686
```suggestion
```